bound/kt/src/main/kotlin/web5/sdk/dids/Document.kt

```diff
@@ -1,8 +1,106 @@
 package web5.sdk.dids
 
-import web5.sdk.rust.DocumentData as RustCoreDocumentData
+import web5.sdk.crypto.keys.Jwk
 
 /**
  * Representation of a [DID Document](https://github.com/TBD54566975/web5-spec/blob/main/spec/did.md).
  */
-typealias Document = RustCoreDocumentData
\ No newline at end of file
+data class Document(
+    val id: String,
+    val context: List<String>?,
```

Comment:
yup the thing we lose is like if we are missing a field or we have something optional and its not optional here or something.

Like Document in theory could get out of sync with RustCoreDocumentData. 

But thats what test vectors and unit tests are for